internal/boxcli/services.go

```diff
@@ -199,6 +205,8 @@ func restartServices(
 		Environment: flags.config.environment,
 		Env:         env,
 		Stderr:      cmd.ErrOrStderr(),
+		// TODO: Should we allow these options to be passed here like we do for service up?
+		ProcessComposeOpts: &devopt.ProcessComposeOpts{Background: true},
```

Comment:
I lean towards only implementing the passthrough flags for up. 

For other commands, we don't always start the process compose server so that flags would be ignored sometimes.